src/concrete/ml/sklearn/rf.py

```diff
@@ -173,7 +173,7 @@ def __init__(
         min_samples_split=2,
         min_samples_leaf=1,
         min_weight_fraction_leaf=0.0,
-        max_features="sqrt",
```

Comment:
😮

Reply:
more than that I think this was a mistake no ? why would the default value for `max_features` be `sqrt` ? 😅

Reply:
No it's actually a proper parameter ^^. It is the default for the classifier but not the regressor.

Reply:
weird naming then haha but ok good to know !